packages/badge/Badge.test.tsx

```diff
@@ -1,18 +1,51 @@
-import { Badge } from './Badge';
-
+import { Badge } from './index';
 import { h, mount } from 'bore';
-import { emit } from 'skatejs';
 
 describe( Badge.is, () => {
 
   it( 'should render', () => {
 
-    console.warn('Missing tests for Badge!');
-
     return mount(
       <Badge/>
-    ).wait();
+    ).wait((element) => element.has('c-badge') );
 
   });
 
+  it( 'should render rounded', () => {
```

Comment:
What do you mean 'nothing'? 😄  ... This test is only for css class with this bool flag. Nothing more...